src/components/common/EffectiveDateTime.vue

```diff
@@ -462,4 +465,18 @@ export default class EffectiveDateTime extends Mixins(DateMixin) {
     color: $BCgovInputError !important;
   }
 }
+::v-deep {
+  .date-picker-form  .v-input__control > .v-input__slot:before {
```

Comment:
This element is not in this component. Should this be moved to the date picker component instead?

Reply:
Same for the v-icon below I think.

Reply:
ok. Will change the DatePicker.